Piwik.Tracker/PiwikTracker.cs

```diff
@@ -585,10 +606,10 @@ protected string GetCookieName(string cookieName)
         /// </summary>
         /// <param name="documentTitle">Page title as it will appear in the Actions > Page titles report</param>
         /// <returns>HTTP Response from the server or null if using bulk requests.</returns>
-        public TrackingResponse DoTrackPageView(string documentTitle = null)
+        public Task<TrackingResponse> TrackPageViewAsync(string documentTitle = null)

```

Comment:
Can you confirm postfixing every method name with "Async" is either
 - required by the compiler, or
 - a naming standard we should follow because developers expect async methods to end with this keyword?